README.txt

```diff
@@ -0,0 +1,31 @@
+# Useful link:
+http://blog.miguelgrinberg.com/post/designing-a-restful-api-with-python-and-flask
+
+# To install flask:
+sudo aptitude install python-flask
+
+# To run some quick tests:
+
+  # load the metaserv schema and load some dummy data
+  ./tests/reinit.sh
+  ./tests.reinit.py
```

Comment:
Yup, done. I actually tried earlier but got stuck, but this time it was a breeze.